apps/web-giddh/src/app/services/ecommerce.service.ts

```diff
@@ -0,0 +1,40 @@
+import { UserDetails } from '../models/api-models/loginModels';
+import { Optional, Inject, Injectable } from '@angular/core';
+import { ErrorHandler } from './catchManager/catchmanger';
+import { HttpWrapperService } from './httpWrapper.service';
+import { ServiceConfig, IServiceConfigArgs } from './service.config';
+import { GeneralService } from './general.service';
+import { ECOMMERCE_API } from './apiurls/shopify.api';
+import { BaseResponse } from '../models/api-models/BaseResponse';
+import { catchError, map } from 'rxjs/operators';
+
+@Injectable()
+export class EcommerceService {
+
+    private user: UserDetails;
+    private companyUniqueName: string;
+
+    constructor(private errorHandler: ErrorHandler, private _http: HttpWrapperService,
```

Comment:
Avoid using '_' in names of dependencies injected or variables as per Angular style guide